<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgExport</class>
 <widget class="QDialog" name="dlgExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Data</string>
  </property>
  <property name="windowIcon">
   <iconset resource="common.qrc">
    <normaloff>:/icon/disk</normaloff>:/icon/disk</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>File Format</string>
   </property>
   <widget class="QRadioButton" name="formatTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TXT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="formatCsv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="formatXml">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>XML</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Source</string>
   </property>
   <widget class="QRadioButton" name="sourceLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log (complete)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sourceFiltered">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log with current filter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sourceCounters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Counters</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Fields</string>
   </property>
   <widget class="QCheckBox" name="fieldDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fieldTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fieldDevice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fieldAscii">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascii data</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fieldHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex data</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fieldSubCounts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Counts per device</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fieldTotalCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total count</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/icon/cross</normaloff>:/icon/cross</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/icon/tick</normaloff>:/icon/tick</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editComment">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections/>
</ui>
